Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -250,19 +250,10 @@ OSD locations defined in the CRUSH map and topology labels on nodes.
 Refer to the [krbd-options document](https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options)
 for more details.
 
-Execute the following step to enable read affinity for a specific ceph cluster:
+Steps to enable the read affinity for a specific ceph cluster:
 
-* Patch the ceph cluster CR to enable read affinity:
-
-```console
-kubectl patch cephclusters.ceph.rook.io <cluster-name> -n rook-ceph -p '{"spec":{"csi":{"readAffinity":{"enabled": true}}}}'
-```
-
-```yaml
-  csi:
-    readAffinity:
-      enabled: true
-```
+* Ensure that `readAffinity.enabled` is set to true in the rook-ceph-csi-config configMap,
+  and provide the necessary crush location labels to `readAffinity.crushLocationLabels`.
```

Comment:
No change is required in this doc right?

Reply:
yes, my bad
reverted it